src/components/list-box/_list-box.scss

```diff
@@ -0,0 +1,242 @@
+//-----------------------------
+// List
+//-----------------------------
+
+@import '../../globals/scss/colors';
+@import '../../globals/scss/css--helpers';
+@import '../../globals/scss/layout';
+@import '../../globals/scss/import-once';
+@import '../../globals/scss/vars';
+
+$list-box-width: rem(300px);
```

Comment:
Question for the group, I know width was a design spec but do we, in general, specify this value? Or no?

Reply:
Usually we don't, for other components we've just done `width: 100%` and then the user can specify the width themselves. We can add in a demo width for when we show it on the website though so it doesn't span all across the container.